packages/interactivity-router/src/head.ts

```diff
@@ -29,6 +37,14 @@ export const updateHead = async ( newHead: HTMLHeadElement[] ) => {
 		}
 	}
 
+	await Promise.all(
+		[ ...headElements.entries() ]
+			.filter( ( [ , { tag } ] ) => tag.nodeName === 'SCRIPT' )
+			.map( async ( [ url ] ) => {
+				await import( url );
```

Comment:
This line was causing problems with DEWP, is that correct? That suggests that webpack is trying to handle this import which is effectively a wildcard. That "URL" could be anything.

Webpack may be trying to bundle up whatever it can find.

This should probably have a `/* webpackIgnore: true */` [magic comment](https://webpack.js.org/api/module-methods/#magic-comments). Webpack should not be handling this import in any way.

Reply:
Nope, this was not the issue. The issue was that there was a cyclical dependency between `index.js` and `head.ts ` in the router package. That was fixed (worked around, really) in https://github.com/WordPress/gutenberg/pull/64067/commits/c7e70ddcf62371a5309626487b672246d42165b7

But you're right that we need the webpack comment here! I've already had it before but temporarily removed it in https://github.com/WordPress/gutenberg/pull/64067/commits/62e527e1769162894881eba76fc33bd96e3fa7eb when trying to debug the problem with cyclical dependencies.